Dockerfile

```diff
@@ -1,17 +1,15 @@
-FROM ruby:2.3-alpine
+FROM ruby:2.4-alpine
+
+RUN adduser -u 9000 -D app
 
 WORKDIR /usr/src/app
-COPY Gemfile /usr/src/app/
-COPY Gemfile.lock /usr/src/app/
 
-RUN gem install bundler && \
-    bundle install -j 4 && \
-    rm -fr /usr/share/ri
+COPY Gemfile Gemfile.lock ./
 
-RUN apk --update add git
+RUN gem install --no-document -g
```

Comment:
~~Nice. Could we expand that option to its full form? `--file`. Weird that `-g` is the shortform. I'd assume it would be short for `--global`.~~

Looks like Gemfile support within the `gem` command has some issues and is not  yet "production-ready": https://github.com/rubygems/rubygems/issues/1207#issuecomment-86158101

> It is possible to use the Gemfile support. It is recommended and currently supported to use Bundler instead.

In particular, it looks like there are some known security vulnerabilities, like this CVE: https://github.com/rubygems/rubygems/issues/1211

Sounds like we should stick with `bundle install` for now.